.github/workflow/npm_deploy.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish package to GitHub Packages
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+      # Setup .npmrc file to publish to GitHub Packages
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org/ntohq'
```

Comment:
@wesdevpro Maybe I am wrong, but don't we need `scope: '@ntohq'` here?

Reply:
> @wesdevpro Maybe I am wrong, but don't we need `scope: '@ntohq'` here?

I believe that is for when you push to the GitHub NPM registry. In this case we are using NPM directly